### PR TITLE
Update README.md task 1

### PR DESCRIPTION
**Pull request by jalas167**

### CODE REVIEW

File: tasks/1-text-2-speech/README.md [hunk 1]
```diff
@@ -44,16 +44,16 @@ Once the generation is complete, an audio file is sent back to the user.
 ### Steps
 
 1. Create a docker image from provided Dockerfile, convert it to a [GVMI](https://handbook.golem.network/requestor-tutorials/vm-runtime/convert-a-docker-image-into-a-golem-image) and publish the generated image to receive an image hash that will be used during the implementation
-2. Modify the `main.mjs` script that will execute the text-2-speech conversion task and will be available under the `npm run tts` command ([Check our docs for introduction and examples](https://docs.golem.network/creators/javascript/high-level/))
+2. Modify the `main.mjs` script that will execute the text-2-speech conversion task and will be available under the `npm run tts` command ([Check our docs for introduction and examples](https://docs.golem.network/docs/creators/javascript/high-level/))
```
Comment (cryptobench) [inline tasks/1-text-2-speech/README.md:47]: link doesnt exist

File: tasks/1-text-2-speech/README.md [hunk 2]
```diff
@@ -44,16 +44,16 @@ Once the generation is complete, an audio file is sent back to the user.
...
 3. Run the script and verify the output 
 4. Add changes in the `text-2-speech` directory
 5. [Fill out the feedback form](./FEEDBACK.md) and publish changes in your repository fork
 
 
 ### Useful links to solve this task
 
-- https://docs.golem.network/creators/javascript/high-level/task-model/
-- https://docs.golem.network/creators/javascript/examples/hello/
+- https://docs.golem.network/docs/creators/javascript/high-level/task-model/
```
Comment (cryptobench) [inline tasks/1-text-2-speech/README.md:55]: doesnt exists

File: tasks/1-text-2-speech/README.md [hunk 3]
```diff
@@ -44,16 +44,16 @@ Once the generation is complete, an audio file is sent back to the user.
...
+- https://docs.golem.network/docs/creators/javascript/examples/hello/
```
Comment (cryptobench) [inline tasks/1-text-2-speech/README.md:56]: doesnt exists

File: tasks/1-text-2-speech/README.md [hunk 4]
```diff
@@ -95,8 +95,8 @@ can also be found at **https://preview.golem.network/**
 
 ## Useful links:
 
-- Javascript docs: https://docs.golem.network/creators/javascript/quickstart/
-- Dapps docs: https://docs.golem.network/creators/dapps/ 
+- Javascript docs: https://docs.golem.network/docs/creators/javascript/quickstart/
```
Comment (cryptobench) [inline tasks/1-text-2-speech/README.md:98]: broken link, should be https://docs.golem.network/docs/creators/javascript/quickstarts/quickstart

**Review by cryptobench — CHANGES_REQUESTED** [review]

**Comment by jalas167** [comment]
will not patch, as mid-level api docs are no longer available, and removing the links would remove the context

**Pull request closed without merging**